From f298e5efd8d2e5a6a4425ec567d88ffb38ea37c4 Mon Sep 17 00:00:00 2001
From: Wayne Zou <b36644@freescale.com>
Date: Wed, 4 Jul 2012 10:01:39 +0800
Subject: [PATCH 337/435] ENGR00182743-2 V4L2 output test: Support
 non-interleaved YUV444 pixel format

Support non-interleaved YUV444 pixel format IPU_PIX_FMT_YUV444P

Signed-off-by: Wayne Zou <b36644@freescale.com>
---
 test/mxc_v4l2_test/mxc_v4l2_output.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/test/mxc_v4l2_test/mxc_v4l2_output.c b/test/mxc_v4l2_test/mxc_v4l2_output.c
index 4884d63..a5bcf45 100644
--- a/test/mxc_v4l2_test/mxc_v4l2_output.c
+++ b/test/mxc_v4l2_test/mxc_v4l2_output.c
@@ -461,6 +461,7 @@ int process_cmdline(int argc, char **argv)
                              (g_in_fmt != V4L2_PIX_FMT_UYVY) &&
 			     (g_in_fmt != V4L2_PIX_FMT_YUYV) &&
 			     (g_in_fmt != V4L2_PIX_FMT_YUV422P) &&
+			     (g_in_fmt != IPU_PIX_FMT_YUV444P) &&
                              (g_in_fmt != V4L2_PIX_FMT_YUV420) &&
                              (g_in_fmt != V4L2_PIX_FMT_NV12) )
                         {
-- 
1.8.0

